grails-app/conf/application.yml

```diff
@@ -123,6 +123,15 @@ security:
         realm: ALA User Details
         path:
           - /status/**
+#ala:
```

Comment:
Shouldn't these be left un-commented for default values?

Reply:
updated